commands/python/libs/k8s/kubernetes.py

```diff
@@ -65,8 +65,8 @@ def kub_exec(self, pod_name, container_name, *command, return_output=False, term
             cmd_list = self._kub_cmd('exec', flags, pod_name, '--', *command)
 
         if return_output:
-            DEVNULL = open(os.devnull, 'w')
-            return subprocess.check_output(cmd_list, stderr=DEVNULL)
+            with open(os.devnull, "w") as devnull:
+                return subprocess.check_output(cmd_list, stderr=devnull)
```

Comment:
Figured I'd fix this while I was in here

Reply:
:+1: :+1: :+1: